src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -64,23 +64,34 @@ public ValueTask DisposeAsync()
         /// <inheritdoc/>
         public void Dispose()
         {
-            this.TelemetryModule.DisposeAsync().AsTask().GetAwaiter().GetResult();
-            this.WebJobsTelemetryModule.DisposeAsync().AsTask().GetAwaiter().GetResult();
+            if (this.TelemetryModule != null)
+            {
+                this.TelemetryModule.DisposeAsync().AsTask().GetAwaiter().GetResult();
+            }
+
+            if (this.WebJobsTelemetryModule != null)
+            {
+                this.WebJobsTelemetryModule.DisposeAsync().AsTask().GetAwaiter().GetResult();
+            }
         }
 
         /// <summary>
         /// Initialize is initialize the telemetry client.
         /// </summary>
         public void Initialize(ILogger logger)
         {
+            this.endToEndTraceHelper = new EndToEndTraceHelper(logger, this.options.Tracing.TraceReplayEvents);
+            this.endToEndTraceHelper.ExtensionWarningAnnouncement(
+                "Durable Functions Distributed Tracing V2 is GA now! Fore more information, please visit, "
+                + "https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-diagnostics?tabs=csharp-inproc#distributed-tracing.");
```

Comment:
Should we update this link to an aka.ms link?

Reply:
Let's use an aka.ms link because it provides more flexibility. It'll be easier to change it to point elsewhere if needed (in case we decide to move the content elsewhere). Aka links also provide metrics, so we can easily see how many people used that link.